piksi_tools/interpolate_event_positions.py

```diff
@@ -0,0 +1,146 @@
+"""
+interpolate_event_trigger.py allows user to analyze log files
+with external event triggers.
+"""
+
+from sbp.client.loggers.json_logger import JSONLogIterator
+from sbp.ext_events import SBP_MSG_EXT_EVENT
+import csv
+
+def lin_interp(o, n, otow, ntow, ttow):
+  """
+  Linearly interpolates distance.
+  
+  Parameters
+  ----------
+  o : integer
+    distance previous to trigger
+  n : integer
+    distance prior to trigger
+  otow : integer
+    TOW of data packet previous to trigger
+  ntow : integer
+    TOW of data packet after trigger
+  ttow : integer
+    TOW of trigger 
+  """
+  d = float(n-o)
+  t = (ntow-otow) 
+  v = d/t
+  return o+(v*(ttow-otow))
+
+
+def write_positions(infile, outfile, msgtype):
+  """
+  Organize and output data to log file.
+
+  Parameters
+  ----------
+  infile : string
+    Log file to get data from.
+  outfile : string
+    Output file.
+  """
+
+  with JSONLogIterator(infile) as log:
+    log = log.next()
+    lasttow = 0
+    deltalasttrigger = 0
+    positiondesplayed = 1
```

Comment:
I think you should rename the position displayed variable.  I don't understand its function.